--- gcc/config/arm/t-slibgcc-iphoneos	2008-08-02 21:28:53.000000000 -0700
+++ gcc/config/arm/t-slibgcc-iphoneos	2011-09-02 23:13:59.844677792 -0700
@@ -51,6 +51,7 @@
 	    cp ./$${mlib}/libgcc_s.$(SHLIB_SOVERSION)$(SHLIB_EXT).tmp \
 	       ./libgcc_s.$(SHLIB_SOVERSION)$(SHLIB_EXT)_T_$${mlib} || exit 1 ; \
 	  done ; \
+          rm ./libgcc_s.$(SHLIB_SOVERSION)$(SHLIB_EXT)_T_v5 ; \
 	  $(LIPO_FOR_TARGET) -output libgcc_s.$(SHLIB_SOVERSION)$(SHLIB_EXT) \
 	    -create libgcc_s.$(SHLIB_SOVERSION)$(SHLIB_EXT)_T* ; \
 	  rm libgcc_s.$(SHLIB_SOVERSION)$(SHLIB_EXT)_T* ; \
