--- gcc/Makefile.in	2011-02-17 16:39:11.000000000 -0800
+++ gcc/Makefile.in	2011-09-02 22:22:49.985056317 -0700
@@ -954,10 +954,10 @@
 # If LLVM is built srcdir != objdir, include the objdir headers for configured
 # headers, like DataTypes.h.
 ifneq ($(LLVMOBJDIR),$(LLVMSRCDIR))
-INCLUDES += -I$(LLVMOBJDIR)/include
+INCLUDES += -isystem $(LLVMOBJDIR)/include
 endif
 
-INCLUDES += -I$(LLVMSRCDIR)/include
+INCLUDES += -isystem $(LLVMSRCDIR)/include
 endif
 # LLVM LOCAL end
 .c.o:
