diff --git a/include/foreign/mach/machine/vm_types.h b/include/foreign/mach/machine/vm_types.h
index 4352ad7..f49754f 100644
--- a/include/foreign/mach/machine/vm_types.h
+++ b/include/foreign/mach/machine/vm_types.h
@@ -3,7 +3,7 @@
 
 #include <stdint.h>
 
-typedef uint32_t        natural_t;
+typedef unsigned long   natural_t;
 typedef int32_t         integer_t;
 
 typedef natural_t       vm_offset_t;
